melissa/actions/spelling.py

```diff
@@ -0,0 +1,10 @@
+# Melissa
+from melissa.tts import tts
+
+
+WORDS = {'repeat_text': {'groups': ['spell']}}
+
+
+def repeat_text(text):
+    text = text.split(' ', 1)[1]
+    tts(text)
```

Comment:
Add a new line above this: `spelling = ' '.join(text)` and change the code in this line to `tts(spelling)`.